angular/projects/spark-angular/src/lib/directives/sprk-button/sprk-button.stories.ts

```diff
@@ -213,6 +213,56 @@ spinningTertiary.parameters = {
   jest: ['sprk-button.directive'],
 };
 
+export const compact = () => ({
+  template: `
+    <button
+      idString="button-compact"
+      sprkButton
+      additionalClasses="sprk-c-Button--compact" 
```

Comment:
This is a directive so you would use just `class` here instead of needing to use the addtionalClasses input.